composition/tests/utils/utils.ts

```diff
@@ -29,12 +29,22 @@ export const versionOneBaseSchema = `
 `;
 
 export const baseDirectiveDefinitions = `
-    directive @extends on INTERFACE | OBJECT
-    directive @external on FIELD_DEFINITION | OBJECT
-    directive @key(fields: openfed__FieldSet!, resolvable: Boolean = true) repeatable on INTERFACE | OBJECT
-    directive @provides(fields: openfed__FieldSet!) on FIELD_DEFINITION
-    directive @requires(fields: openfed__FieldSet!) on FIELD_DEFINITION
-    directive @tag(name: String!) repeatable on ARGUMENT_DEFINITION | ENUM | ENUM_VALUE | FIELD_DEFINITION | INPUT_FIELD_DEFINITION | INPUT_OBJECT | INTERFACE | OBJECT | SCALAR | UNION
+  directive @extends on INTERFACE | OBJECT
+  directive @external on FIELD_DEFINITION | OBJECT
+  directive @key(fields: openfed__FieldSet!, resolvable: Boolean = true) repeatable on INTERFACE | OBJECT
+  directive @provides(fields: openfed__FieldSet!) on FIELD_DEFINITION
+  directive @requires(fields: openfed__FieldSet!) on FIELD_DEFINITION
+  directive @tag(name: String!) repeatable on ARGUMENT_DEFINITION | ENUM | ENUM_VALUE | FIELD_DEFINITION | INPUT_FIELD_DEFINITION | INPUT_OBJECT | INTERFACE | OBJECT | SCALAR | UNION
+`;
+
+export const baseDirectiveDefinitionWithConfigureDescription = `
+  directive @extends on INTERFACE | OBJECT
+  directive @external on FIELD_DEFINITION | OBJECT
+  directive @key(fields: openfed__FieldSet!, resolvable: Boolean = true) repeatable on INTERFACE | OBJECT
+  directive @openfed__configureDescription(federatedGraphDescriptionOverride: String, propagateToFederatedGraph: Boolean! = true) on ARGUMENT_DEFINITION | ENUM | ENUM_VALUE | FIELD_DEFINITION | INPUT_FIELD_DEFINITION | INPUT_OBJECT | INTERFACE | OBJECT | SCALAR | SCHEMA | UNION
```

Comment:
I'd suggest a shorter name. What about `description` and `propagate`? Documentation will help to understand the inner workings.

Reply:
We can rename them no problem, but these were the clearest names I could think of.

Reply:
How about `propagate` and `descriptionOverride`?